ingestion-edge/constraints.txt

```diff
@@ -53,5 +53,5 @@ urllib3==1.24.1 --hash=sha256:61bf29cada3fc2fbefad4fdf059ea4bd1b4a86d2b6d15e1c7c
 urllib==1.23 --hash=sha256:b5725a0bd4ba422ab0e66e89e030c806576753ea3ee08554382c14e685d117b5
 uvloop==0.12.1 --hash=sha256:c684047c6cf6d697ba37872fb1b4489012ea91f3f802c8fbb9c367c4902e88dc --hash=sha256:a585781443eeb2edb858f8c08c503aac237a5f1bebf0c84ea8340cc337afa408 --hash=sha256:9cfdb966ae804c46b96c92207dfd2174935ffc70e706e42e1c94c60d16dbe860 --hash=sha256:435e232869923fd2248e4ca0ad73e24a5b4debf40bed9dcde133cfe1bef98a7a --hash=sha256:da5a59d8812188b57b5783c7fb78891d14dd1050b6259680e0dbd4253d7d0f64 --hash=sha256:1c47f197be8f0a3c651dd20be1e1bd43268186246f246d4e86c91e95a89e4865 --hash=sha256:1c175f47d34b84e33c0e312f4987c927ea004afc3a5f05d2f0f610d71d0e4c89 --hash=sha256:3fd4943570d20e8cd4d9f0a3190ebd5cf040e5610b685e05c878128a11f7ad14 --hash=sha256:b296493e033846e46488a6aa227a75c790091f5ee5456ec637bb0badad1e8851 --hash=sha256:198fe0c196056930ec6c4a0a878e531a66d15467ca7c74a875aa90271f0c6e3f
 websocket-client==0.54.0 --hash=sha256:8c8bf2d4f800c3ed952df206b18c28f7070d9e3dcbd6ca6291127574f57ee786 --hash=sha256:e51562c91ddb8148e791f0155fdb01325d99bb52c4cdbb291aee7a3563fd0849
-websockets==6.0 --hash=sha256:669d1e46f165e0ad152ed8197f7edead22854a6c90419f544e0f234cc9dac6c4
+websockets==6.0 --hash=sha256:0e2f7d6567838369af074f0ef4d0b802d19fa1fee135d864acc656ceefa33136 --hash=sha256:2a16dac282b2fdae75178d0ed3d5b9bc3258dabfae50196cbb30578d84b6f6a6 --hash=sha256:5a1fa6072405648cb5b3688e9ed3b94be683ce4a4e5723e6f5d34859dee495c1 --hash=sha256:5c1f55a1274df9d6a37553fef8cff2958515438c58920897675c9bc70f5a0538 --hash=sha256:669d1e46f165e0ad152ed8197f7edead22854a6c90419f544e0f234cc9dac6c4 --hash=sha256:695e34c4dbea18d09ab2c258994a8bf6a09564e762655408241f6a14592d2908 --hash=sha256:6b2e03d69afa8d20253455e67b64de1a82ff8612db105113cccec35d3f8429f0 --hash=sha256:79ca7cdda7ad4e3663ea3c43bfa8637fc5d5604c7737f19a8964781abbd1148d --hash=sha256:7fd2dd9a856f72e6ed06f82facfce01d119b88457cd4b47b7ae501e8e11eba9c --hash=sha256:82c0354ac39379d836719a77ee360ef865377aa6fdead87909d50248d0f05f4d --hash=sha256:8f3b956d11c5b301206382726210dc1d3bee1a9ccf7aadf895aaf31f71c3716c --hash=sha256:91ec98640220ae05b34b79ee88abf27f97ef7c61cf525eec57ea8fcea9f7dddb --hash=sha256:952be9540d83dba815569d5cb5f31708801e0bbfc3a8c5aef1890b57ed7e58bf --hash=sha256:99ac266af38ba1b1fe13975aea01ac0e14bb5f3a3200d2c69f05385768b8568e --hash=sha256:9fa122e7adb24232247f8a89f2d9070bf64b7869daf93ac5e19546b409e47e96 --hash=sha256:a0873eadc4b8ca93e2e848d490809e0123eea154aa44ecd0109c4d0171869584 --hash=sha256:cb998bd4d93af46b8b49ecf5a72c0a98e5cc6d57fdca6527ba78ad89d6606484 --hash=sha256:e02e57346f6a68523e3c43bbdf35dde5c440318d1f827208ae455f6a2ace446d --hash=sha256:e79a5a896bcee7fff24a788d72e5c69f13e61369d055f28113e71945a7eb1559 --hash=sha256:ee55eb6bcf23ecc975e6b47c127c201b913598f38b6a300075f84eeef2d3baff --hash=sha256:f1414e6cbcea8d22843e7eafdfdfae3dd1aba41d1945f6ca66e4806c07c4f454
```

Comment:
added missing osx hashes for two dependencies, so that `pip install -r` works there